### PR TITLE
TrackerSoftware.PDF-XChangeStandard version 9.0.354.0

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19471)

### CODE REVIEW

File: manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TrackerSoftware.PDF-XChangeStandardPrinter
+PackageVersion: 9.0.354.0
+PackageLocale: en-US
+Publisher: Tracker Software
+PackageName: PDF-XChange Standard Printer
```
Comment (OfficialEsco) [inline manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.locale.en-US.yaml:8]:
```suggestion
Publisher: Tracker Software Products (Canada) Ltd.
PackageName: PDF-XChange Standard
```

File: manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TrackerSoftware.PDF-XChangeStandardPrinter
```
Comment (OfficialEsco) [inline manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.installer.yaml:4]:
```suggestion
PackageIdentifier: TrackerSoftware.PDF-XChangeStandard
```

File: manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TrackerSoftware.PDF-XChangeStandardPrinter
```
Comment (OfficialEsco) [inline manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TrackerSoftware.PDF-XChangeStandard
```

File: manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TrackerSoftware.PDF-XChangeStandardPrinter
```
Comment (OfficialEsco) [inline manifests/t/TrackerSoftware/PDF-XChangeStandardPrinter/9.0.354.0/TrackerSoftware.PDF-XChangeStandardPrinter.yaml:4]:
```suggestion
PackageIdentifier: TrackerSoftware.PDF-XChangeStandard
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to change the folder and file name.

**Comment by krystofkrticka** [comment]
Working on it.

**Comment by OfficialEsco** [comment]
`Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 'TrackerSoftware.PDF-XChangeStandardPrinter.installer.yaml', expected 'TrackerSoftware.PDF-XChangeStandard.installer.yaml'`
((:

**Comment by krystofkrticka** [comment]
I know already fixing it.

**Comment by krystofkrticka** [comment]
Hopefully it didn't gets messed up since I was testing changing it with remote repositories plugin in VSCode

**Comment by krystofkrticka** [comment]
> Hopefully it didn't gets messed up since I was testing changing it with remote repositories plugin in VSCode

Well it got messed up, rename in remote repositories for VS Code works differently than I thought. I thought that it would delete automatically files with old name like it happen with classic command line git when you change file names in preferred file manager.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**